scio-test/src/test/scala/com/spotify/scio/testing/JobTestTest.scala

```diff
@@ -841,4 +895,106 @@ class JobTestTest extends PipelineSpec {
     }
     e.getMessage should endWith(" was not greater than or equal to 100")
   }
+
+  "transformOverride" should "pass with a source override" in {
+    JobTest[SourceTransformOverrideJob.type]
+      .args("--input=in.txt", "--output=out.txt")
+      .input(TextIO("in.txt"), Seq("1", "2", "3"))
```

Comment:
Unless I'm missing smth there's really not point in setting `input` here ?

Reply:
I see. I think we can live with that for now. I expect most people won't override "normal" IOs so they should not run into that. Assuming we replace the current `JobTest` implementation with overrides in the future, that problem will be solved.